draft-ietf-quic-transport.md

```diff
@@ -5495,6 +5497,24 @@ Reason Phrase:
   This SHOULD be a UTF-8 encoded string {{!RFC3629}}.
 
 
+## HANDSHAKE_DONE frame {#handshake-done-frame}
+
+The server uses the HANDSHAKE_DONE frame (type=0x1e) to signal confirmation of
+the handshake to the client.  The HANDSHAKE_DONE frame contains no additional
+fields.
+
+This frame can only be sent by the server. Servers MUST NOT send a
+HANDSHAKE_DONE frame before completing the handshake.  A server MUST treat
+receipt of a HANDSHAKE_DONE frame as a connection error of type
+PROTOCOL_VIOLATION.
+
+A client MAY consider the handshake to be confirmed when it receives an
+acknowledgement for a 1-RTT packet.  This can be implemented by recording
+the lowest packet number sent with 1-RTT keys, and the highest value of the
+Largest Acknowledged field in any received 1-RTT ACK frame: once the latter is
+higher than or equal to the former, the handshake can be confirmed.
```

Comment:
Sorry about catching this so late, but this is entirely editorial: This para does not belong here; this section is the frame description section. Can you move this para to the {{handshake-confirmed}} section in QUIC-TLS?

Reply:
Gah.  I had the same thought before but failed to write it down.  Thanks for catching that.  I'll open a PR.